plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Purchase.php

```diff
@@ -54,20 +54,52 @@ public function get_id() {
 	 * @return string
 	 */
 	public function get_title() {
-		$products = $this->get_paid_products_and_themes();
+		$products   = $this->get_paid_products_and_themes();
+		$first_product    = count( $products['remaining'] ) >= 1 ? $products['remaining'][0] : false;
+		$additional_count = count( $products['remaining'] ) - 1;
+
+		if ( $this->get_parent_option( 'use_completed_title' ) && $this->is_complete() ) {
+			return count( $products['remaining'] ) === 1
+				? sprintf(
+					/* translators: %1$s: a purchased product name */
+					__(
+						'You added %1$s',
+						'woocommerce'
+					),
+					$first_product
+				)
+				: sprintf(
+					/* translators: %1$s: a purchased product name, %2$d the number of other products purchased */
+					_n(
+						'You added %1$s and %2$d other product',
+						'You added %1$s and %2$d other products',
+						$additional_count,
+						'woocommerce'
+					),
+					$products['remaining'][0],
```

Comment:
Also in the $products['purchaseable'] the items are array's of plugins, so this will probably be `$products['purchaseable'][0]['label']`